### PR TITLE
Make @help use a public datasource

### PR DESCRIPTION
**Pull request by albandum**
## Description

Make @help use a public datasource

### CODE REVIEW

**Comment by spolu** [comment]
Not sure I understand the purpose here? Is it to give helper access to data sources?

**Comment by flvndvd** [comment]
> Not sure I understand the purpose here? Is it to give helper access to data sources?

We want to use a public data source for the `@help` so that everyone in the GTM/CS team can add new resources. We found limitations on the current approach that relies on a websearch.

**Comment by spolu** [comment]
Roger that

File: front/lib/api/assistant/global_agents.ts [hunk 1]
```diff
@@ -169,13 +173,23 @@ function _getHelperGlobalAgent({
...
+        topK: "auto",
+        dataSources: [
+          {
+            dataSourceViewId: PRODUCTION_DUST_APPS_HELPER_DATASOURCE_VIEW_ID,
+            workspaceId: PRODUCTION_DUST_APPS_WORKSPACE_ID,
+            filter: { parents: null },
+          },
+        ],
+        name: "search_dust_docs",
+        description: `The documentation of the Dust platform.`,
```
Comment (flvndvd) [inline front/lib/api/assistant/global_agents.ts:189]: `nit`
```suggestion
        description: "The documentation of the Dust platform.",
```

**Review by flvndvd — APPROVED** [review]
LGTM

**Pull request merged**